### ISSUE TITLE
missing nexus-maven-repository-index.properties file in http://oss.jfrog.org/artifactory/oss-snapshot-local/ and http://oss.jfrog.org/simple/oss-snapshot-local/io/springfox/

### ISSUE THREAD

**Issue by abelyaevskiy** [springfox/springfox#1048]
Hi!

I just tried to setup my project with springfox 2.2.3 snapshot version in Intellij IDEA 14 and failed with error that nexus-maven-repository-index.properties file missed in both http://oss.jfrog.org/artifactory/oss-snapshot-local/ and http://oss.jfrog.org/simple/oss-snapshot-local/io/springfox/ repositories. I use these repos because of instruction http://springfox.github.io/springfox/docs/snapshot/.

**Comment by dilipkrish** [comment]
@abelyaevskiy not sure what this problem is. Seems like a maven/nexus issue on your setup? It works totally as expected when I try to use it.

**Comment by abelyaevskiy** [comment]
@dilipkrish Hi! It's my bad. 

I found issues with dependency donwloading:
 * when I work from corporate network I have SSL certificate issue
 * initially I added `maven { url 'http://oss.jfrog.org/artifactory/oss-snapshot-local/' }` into `buildscript {...}` section, I changed it to `repositories {...}` and fixed the issue
 * also I updated my gradle to 2.4 version (but it didn't help)

Now my starting issue is fixed.
